### PR TITLE
Revive quote report

### PR DESCRIPTION
**Pull request by PirosB3**
Fixes Quote Report so that is *only* reports the following:
- All RFQ sources “considered”
- The final sources “delivered” (the order salad delivered in the quote)

This should result in < 1% off the existing logging while providing the same value as before

### CODE REVIEW

File: packages/asset-swapper/src/utils/quote_report_generator.ts [hunk 1]
```diff
@@ -65,25 +66,17 @@ export interface QuoteReport {
  */
 export function generateQuoteReport(
     marketOperation: MarketOperation,
-    dexQuotes: DexSample[],
-    multiHopQuotes: Array<DexSample<MultiHopFillData>>,
+    _dexQuotes: DexSample[],
```
Comment (alexkroeger) [inline packages/asset-swapper/src/utils/quote_report_generator.ts:69]: can we just get rid of these arguments: `dexQuotes` and `multiHopQuotes`?

File: packages/asset-swapper/test/quote_report_generator_test.ts [hunk 2]
```diff
@@ -58,32 +58,18 @@ describe('generateQuoteReport', async () => {
     it('should generate report properly for sell', () => {
         const marketOperation: MarketOperation = MarketOperation.Sell;
 
-        const kyberSample1: DexSample = {
-            source: ERC20BridgeSource.Kyber,
-            input: new BigNumber(10000),
-            output: new BigNumber(10001),
-            fillData: {},
-        };
         const kyberSample2: DexSample = {
```
Comment (phil-ociraptor) [inline packages/asset-swapper/test/quote_report_generator_test.ts:61]: super nit: now that `kyberSample1` is removed, can this just be named `kyberSample`. Same for uni below

**Review by phil-ociraptor — APPROVED** [review]
Looks good on my side! I don't entirely understand the removal of the two hop and multihop stuff, but overall, my understanding is this should reduce the amount of data that gets logged

File: packages/asset-swapper/src/utils/quote_report_generator.ts [hunk 3]
```diff
@@ -44,6 +44,7 @@ export interface NativeRfqOrderQuoteReportEntry extends QuoteReportEntryBase {
     fillData: NativeFillData;
     fillableTakerAmount: BigNumber;
     isRfqt: true;
+    nativeOrder: RfqOrderFields;
```
Comment (alexkroeger) [inline packages/asset-swapper/src/utils/quote_report_generator.ts:47]: no similar adjustment needed for limit orders?

**Review by alexkroeger — APPROVED** [review]

**Pull request merged**